README.md

```diff
@@ -4,17 +4,17 @@ Inspired by the [it takes a village](https://verifa.io/blog/the-verifa-community
 
 ## Why Tabete exists
 
-We're a circle of hands-on consultants who know that sharing time together will make us better at our craft.
-Having done regular check-ups with other consultants for many years, we've noticed a couple of things:
+We're a circle of software development consultants who know that sharing time together will hone our craft.
+Having met regularly to talk shop for many years, we've noticed a couple of things:
 
 ### Osmosis is real
 
-Simply being around a group of peers lets ideas, perspectives and knowledge cross-pollinate. The conversations in our meetings or when one shows something they've done for their client aren't just theoretical; they spark inspiration and encourage us to explore how to implement those ideas in our own clients' environments.
+Simply being around peers lets ideas, perspectives and knowledge spread. The conversations and showcases in our meetings aren't just theoretical; they spark inspiration and allow us to bring fresh ideas to our own clients' environments.
```

Comment:
The reason I like "cross-pollinate" it's because it describes a two-way street (following the feel of osmosis). "Spreading knowledge" feels one-way.

Maybe go with something like "exchanging knowledge"? Or just keep cross-pollinate?

Reply:
Fair point, brought back cross-pollinate and updated the phrasing on encouraging exploration